pynicotine/gtkgui/widgets/theme.py

```diff
@@ -260,6 +260,9 @@ def set_global_style(isolated_mode=False):
 PRIVATE_ICON_LABELS = {
     "changes-prevent-symbolic": _("Private")
 }
+READABLE_ICON_LABELS = {
+    "dialog-warning-symbolic": _("Unreadable")
```

Comment:
Consider that this same column could be used to indicate the presence of other kinds of errors logged during the actual rescan process of Scanner() and maybe a tick or something for a successfully completed rescan. Suggest call it `FOLDER_ICON_LABELS`.